cmd/transcriber/call/tracks.go

```diff
@@ -115,6 +116,21 @@ func (t *Transcriber) processLiveTrack(track trackRemote, sessionID string, user
 	}
 	defer oggWriter.Close()
 
+	// Live captioning:
+	// pktPayloadCh is used to send the rtp audio data to the processLiveCaptionsForTrack goroutine
+	var pktPayloadCh chan []byte
+	if t.cfg.LiveCaptionsOn {
+		pktPayloadCh = make(chan []byte, pktPayloadChBuffer)
+		doneChan := make(chan struct{})
+		defer func() {
+			close(pktPayloadCh)
+			close(doneChan)
+		}()
```

Comment:
Wondering if we could use the single `pktPayloadCh` to track whether we are done or not.

Reply:
Yep that works, good suggestion, thanks.